web/pageAdmin.go

```diff
@@ -165,7 +165,16 @@ func (s *webServer) handlerAdminUserEdit(w http.ResponseWriter, r *http.Request)
 
 		return
 	case "ban":
-		s.backend.AdminBanUser(user)
+		err = s.backend.AdminBanUser(user)
+		if err != nil {
+			if err != nil {
```

Comment:
Double err checks here

Reply:
That was just me being stupid lul, i'll fix that tomorrow

Reply:
Bumping this so i can merge this hopefully soon.